.circleci/wait.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+wait_for_cassandra () {
+  for i in 1 2 3 4 5; do
```

Comment:
`i` looks unused, consider instead:

```suggestion
  for (( i=0; i<5; ++i )); do
```

similar below